AI/Nullkiller/Engine/FuzzyEngines.cpp

```diff
@@ -81,7 +81,7 @@ armyStructure evaluateArmyStructure(const CArmedInstance * army)
 
 		vstd::amax(maxSpeed, creature->valOfBonuses(selectorSTACKS_SPEED, keySTACKS_SPEED));
 	}
-	armyStructure as;
+	armyStructure as{};
```

Comment:
IMO instead of this `armyStructure` should have constructor.
This is not self-descriptive and requires understanding of initialization types.
It is also easy to accidentally end up with uninitialized structure simply because of missing brackets.